### PR TITLE
Proposed .gitattributes file.

### PR DESCRIPTION
**Pull request by bbannon**
Make all text files end-of-line = LF on check-in and check-out.
Eliminate the need to "remove annoying CR" in the code.

Use ident to identify the file.
Similar to
// $Id: view_w.php,v 1.81 2009/11/22 22:26:18 bbannon Exp $
on SourceForge/CVS.
except it's a just 40 char hash between $id: and ending $.

I don't know why both files show up here when I committed them on different branches.
And git status only showed .gitattributes before add/commit/push.

### CODE REVIEW

File: view_w.php [hunk 1]
```diff
@@ -39,7 +39,7 @@
 $nextStr = translate ( 'Next' );
 $prevStr = translate ( 'Previous' );
 
-$can_add = ( empty ( $ADD_LINK_IN_VIEWS ) || $ADD_LINK_IN_VIEWS != 'N' );
+$can_add = ( $ADD_LINK_IN_VIEWS === 'Y' );
```
Comment (craigk5n) [inline view_w.php]: I would remove this change from the pull request since it is related to another pull request.

Comment (bbannon) [inline view_w.php]: I've made so many changes, I lose track of a few. Although, I'm getting better at it.
How would I remove that change? Which pull request is it related to?
For that matter, is there a way to cancel a pull request altogether?

Comment (bbannon) [inline view_w.php]: Speaking of the
// $Id: view_w.php,v 1.81 2009/11/22 22:26:18 bbannon Exp $
line. Since GitHub doesn't track version numbers, and doesn't expand $Id$ into anything useful, is there any point in keeping these lines in the code?

Comment (craigk5n) [inline view_w.php]: Instead of $Id$ we usually use the commit id.  If you do a git log on a particular file, you will see the history and commit ids.  So, I don't see much point to retaining the old CVS $Id$ stuff.  To remove you could just revert the change and commit it as it was before.  I believe that changes is related to the one that switches to using '==='.
